src/ui.cpp

```diff
@@ -289,6 +289,7 @@ void uilist::init()
     max_column_len = 0;      // for calculating space for second column
 
     input_category = "UILIST";
+    input_context ctxt( input_category, keyboard_mode::keychar );
```

Comment:
This only creates an input context in the local scope, which means the `ctxt` member still uses keycode mode, which breaks text input in keycode mode.